packages/react/src/utilities/RovingFocus/RovingFocusRoot.tsx

```diff
@@ -21,6 +21,18 @@ type RovingFocusRootBaseProps = {
    * @default false
    */
   asChild?: boolean;
+  /**
+   * If `true`, the `RovingFocusItem` will be focusable with the left and right arrow keys.
+   *
+   * @default false
+   */
+  verticalArrows?: boolean;
+  /**
+   * If `true`, the `RovingFocusItem` will be focusable with the up and down arrow keys.
+   *
+   * @default true
+   */
+  horizontalArrows?: boolean;
```

Comment:
Aren't these mutually exclusive? 🤔  

Suggest using a string literal for this instead; `orientation: 'vertical' | 'horizontal'`

Reply:
I don't believe they are, check out this; https://ui.shadcn.com/docs/components/toggle-group

Reply:
Either way we should still avoid multiple flags for the same thing as it leads to more complex bloated api and using multiple flags is often harder to maintain/read. 

To support both cases we can do something like `orientation: 'vertical' | 'horizontal' | 'ambiguous'` and then just change the switch case to check on the `orientation` prop. 

Then just check for left/right or up/down in the respective cases, like so; `if (['ArrowLeft', 'ArrowRight'].includes(e.key)) {…}`